eng/scripts/Verify-And-Merge-PRs.ps1

```diff
@@ -1,29 +1,10 @@
 param(
   $PRDataArtifactPath,
   $AuthToken,
-  $ShouldMerge,
-  [switch]$devOpsLogging = $false
+  $ShouldMerge
 )
 
-function LogWarning
-{
-  if ($devOpsLogging) {
-    Write-Host "##vso[task.LogIssue type=warning;]$args"
-  }
-  else {
-    Write-Warning "$args"
-  }
-}
-
-function LogError
-{
-  if ($devOpsLogging) {
-    Write-Host "##vso[task.logissue type=error]$args"
-  }
-  else {
-    Write-Error "$args"
-  }
-}
+. "${PSScriptRoot}..\common\scripts\logging.ps1"
```

Comment:
nit: PowerShell conventions would make this ```Logging.ps1```

Reply:
Renaming in git where the only change is a case change ends up causing more troubles than benefit. I'll just leave this.

Reply:
Yes we can potentially clean some of this up with https://github.com/Azure/azure-sdk-tools/issues/829.